cmd/pipeline/main.go

```diff
@@ -223,7 +225,7 @@ func main() {
 		logger.Warn("configuration file not found")
 	}
 
-	err = config.Validate()
+	err = errors.WrapIf(config.Validate(), "invalid configuration")
```

Comment:
In the new config the returned error is a combined error. Please check that this change works with combined errors (multiple configuration validations).